homeassistant/components/netatmo/__init__.py

```diff
@@ -62,17 +60,14 @@
 _LOGGER = logging.getLogger(__name__)
 
 CONFIG_SCHEMA = vol.Schema(
-    vol.All(
-        cv.deprecated(DOMAIN),
-        {
-            DOMAIN: vol.Schema(
-                {
-                    vol.Required(CONF_CLIENT_ID): cv.string,
-                    vol.Required(CONF_CLIENT_SECRET): cv.string,
-                }
-            )
-        },
-    ),
+    {
```

Comment:
Is this change on purpose? or does it come from a bad rebase?
It feels like this is reverting #71884

cc @allenporter